pkg/api/licenses.go

```diff
@@ -45,6 +45,8 @@ import (
 //	@Param			page			query		int						false	"Page number"
 //	@Param			limit			query		int						false	"Limit of responses per page"
 //	@Param			externalRef		query		string					false	"External reference parameters"
+//	@Param			sort_by			query		string					false	"string enums"	Enums(rf_spdx_id, rf_shortname, rf_fullname)	"Sort by rf_spdx_id, rf_shortname or rf_fullname"
+//	@Param			order_by		query		string					false	"string enums"	Enums(asc, desc)								"Order by asc or desc"
```

Comment:
Description should come before modifiers. Also, default values can also be set :-)

```suggestion
//	@Param			sort_by			query		string					false	"Sort by field"	Enums(rf_spdx_id, rf_shortname, rf_fullname)	default(rf_shortname)
//	@Param			order_by		query		string					false	"Asc or desc ordering"	Enums(asc, desc)	default(asc)
```